gisTrainingNotes.txt

```diff
@@ -0,0 +1,125 @@
+GIS SPECIALIST Developer Education Track
```

Comment:
we'll make a real bcgov repo with education materials after, i used this one because you all are set up to work on it already